tailwind-workspace-preset.js

```diff
@@ -5,6 +5,7 @@ module.exports = {
     extend: {
       spacing: {
         'right-help-sidebar': '22.5rem',
+        'page-container': 'calc(100vh - 64px)',
```

Comment:
mmh it's not exaclty what I wanted but that will do the job. I wanted that we have a variable called `header-height: 64px` and in the code I wanted that you put on the header `h-header-height` and then I wanted the page-container height to be calculated like `calc(100vh - header-height)`
Like this, if we change the header-height in the future, everything will reset itself perfectly

Reply:
I understand ! I don't know if it work like that but I try